packages/cli/src/commands/alias/ls.ts

```diff
@@ -22,13 +23,19 @@ export default async function ls(
 ) {
   const { output } = client;
   const { '--next': nextTimestamp } = opts;
+  const { '--limit': limit } = opts;
   const { contextName } = await getScope(client);
 
   if (typeof nextTimestamp !== undefined && Number.isNaN(nextTimestamp)) {
     output.error('Please provide a number for flag --next');
     return 1;
   }
 
+  if (typeof limit === 'number' && (Number.isNaN(limit) || limit > 100)) {
```

Comment:
Probably should also check that limit > 0.

Reply:
Yeah and maybe also decimals. The API is actually interesting what it does with the input:

```
URL: https://api.vercel.com/v3/now/aliases?limit=-5
Status Code: 200
Results: 1
URL: https://api.vercel.com/v3/now/aliases?limit=0
Status Code: 200
Results: 10
URL: https://api.vercel.com/v3/now/aliases?limit=0.1
Status Code: 200
Results: 1
URL: https://api.vercel.com/v3/now/aliases?limit=9.1
Status Code: 500
Error: An unexpected internal error occurred
```

Mostly I was just trying to catch and help the user, with what felt like a common use case "of the limit isn't enough and I don't want to do pagination, let's try increasing it".

Reply:
Thank you for the feedback @cb1kenobi, I've added all these checks into the function.